src/pages/distroSettings/index.tsx

```diff
@@ -73,10 +83,19 @@ const DistroSettings: React.VFC = () => {
         </SideNavGroup>
       </SideNav>
       <PageWrapper data-cy="distro-settings-page">
-        {!loading && data.distro && <DistroSettingsTabs distro={data.distro} />}
+        {!loading && data?.distro && (
+          <DistroSettingsTabs distro={data.distro} />
+        )}
       </PageWrapper>
     </DistroSettingsProvider>
   );
 };
 
+const ButtonsContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: ${size.xs};
+  margin: 0 ${size.s};
```

Comment:
I like your `ButtonContainer` better 😛 but using LG's `sideNavItemSidePadding` may be a good improvement! https://github.com/evergreen-ci/spruce/pull/1966/files#diff-9a330e784ff5724935be4ccaf76daac3e0986c81c8f6bcff59b6266ad94903e7